lib/chef/provider/subversion.rb

```diff
@@ -194,6 +196,21 @@ def authentication
         result
       end
 
+      def proxy
+        %w{http https}.each { |p|
+          if @new_resource.repository.start_with?("#{p}:") &&
+              ENV["#{p}_proxy"] != nil &&
```

Comment:
The Chef::Config proxy values will get exported, so we want to pick them up from env (to handle the case where chef config does not have a specified value).